### ISSUE TITLE
Make the doxygen search input stylable

### ISSUE THREAD

**Issue by mosra** [mosra/m.css#60]
Things piled up and I'm unable to look at them right now, so opening an issue. The Doxygen theme should have the search input focus outline color configurable, something like this:

```css
.m-dox-search input:focus {
   outline-color: <something>;
}
```

Things to consider:

- which color from the palette to use?
- make the outline configurable for the whole of m.css? (for example I might use that for activated canvases on http://magnum.graphics/showcase/)
- make it possible to configure also outline width etc. (e.g. the outline around this textarea on github is thicker and it looks nice)

Cc: @jbakosi

**Comment by jbakosi** [comment]
> which color from the palette to use?

I have used #a5c9ea from the blue/dark theme (and #a5eac9 for a green-dark one).

**Comment by jbakosi** [comment]
Ah man! I could totally visualize that based on the hex values!

(Hehe, totally kidding of course.)

This is very cool. Thanks. ;-)
